gneiss/balances.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import division
+import numpy as np
+from skbio.stats.composition import clr_inv
+from collections import OrderedDict
+
+
+def _balance_basis(tree_node):
+    """ Helper method for calculating balance basis
+    """
+
+    n_tips = sum([n.is_tip() for n in tree_node.traverse()])
+    counts = _count_matrix(tree_node)
+    counts = OrderedDict([(x, counts[x])
```

Comment:
Ok, I actually take that back.

The reason why I'm doing the OrderedDict calculation outside of `_count_matrix` was to ease the process of testing `_count_matrix`.  I'm reverting back the change - hope that is ok.